scripts/ci_deploy.sh

```diff
@@ -11,4 +11,4 @@ cf -v
 cf login -a https://api.system.staging.digital.gov.au -o dto -u $CF_USER_STAGING -p $CF_PASSWORD_STAGING
 cf target -o dto -s digital-marketplace
 cf push -c 'python application.py db upgrade' -u 'none'
```

Comment:
You should add an -i flag here, so that we still run the migration with one instance even if we scale up the main application.  Otherwise all looks good to me.